src/main/java/com/minecolonies/colony/workorders/AbstractWorkOrder.java

```diff
@@ -34,14 +48,58 @@
 
     /**
      * Default constructor; we also start with a new id and replace it during loading;
-     * this greatly simplifies creating subclasses
+     * this greatly simplifies creating subclasses.
      */
     public AbstractWorkOrder()
     {
         //Should be overridden
     }
 
     /**
+     * Setter for the priority.
+     * @param priority the new priority.
+     */
+    public void setPriority(int priority)
+    {
+        this.priority = priority;
+    }
+
+    /**
+     * Getter for the priority.
+     */
+    public int getPriority()
+    {
+        return this.priority;
+    }
+
+
+    /**
+     * Checks if the workOrder has changed.
+     * @return true if so.
+     */
+    public boolean hasChanged(){ return changed; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)